src/NuGet.Core/NuGet.Client/ManagedCodeConventions.cs

```diff
@@ -139,7 +142,7 @@ private static object Locale_Parser(string name, PatternTable table)
             return null;
         }
 
-        private static object TargetFrameworkName_Parser(
+        private object TargetFrameworkName_Parser(
```

Comment:
I wasn't able to find a scenario where the content model is shared across threads. This class is unique per graph and used while building the lock file and for compatibility checks, both of which happen as final steps in restore.